### PR TITLE
fix naming: ft.Service -> ft.Middleware

### PR DESCRIPTION
**Pull request by christain-rogobete**

### CODE REVIEW

File: doc/appendix-de-kassensichv/reference-tables/reference-tables.md [hunk 1]
```diff
@@ -279,3 +279,474 @@ For definitions regarding national laws, please refer to the appropriate appendi
 
 *Table 15. Type of Signature: ftSignatureType*
 
+### DSFinV-K export
```
Comment (TSchmiedlechner) [inline doc/appendix-de-kassensichv/reference-tables/reference-tables.md:282]: Should these changes be part of this PR, or of #56?

File: doc/appendix-de-kassensichv/terminology/terminology.md [hunk 2]
```diff
@@ -2,7 +2,16 @@
 
 | term | description |
 |---|---|
-| Action (DE: Vorgang) | an "action" within the meaning of the KassenSichV is a related recording process of an electronic recording system. An action can comprise one or more business-actions as well as other procedures, occurrences and events. Each action must trigger at least one transaction logged by the certified technical safety device (DE: TSE).  |
+| Action (DE: Vorgang) | an "action" within the meaning of the KassenSichV is a related recording process of an electronic recording system. An action can comprise one or more business-actions as well as other procedures, occurrences and events (DE: andere Vorgänge). Each action must trigger at least one transaction logged by the certified technical safety device (DE: TSE).  |
```
Comment (TSchmiedlechner) [inline doc/appendix-de-kassensichv/terminology/terminology.md:5]: Same as above; Should these changes be part of this PR, or of #56?

File: doc/appendix-de-kassensichv/reference-tables/reference-tables.md (hunk 1 above)
Comment (christain-rogobete) [reply doc/appendix-de-kassensichv/reference-tables/reference-tables.md]: Hi Tom, true, thank you for the hint. I removed the dsfivk related changes. Can you pls. check it again?

File: doc/appendix-de-kassensichv/terminology/terminology.md (hunk 2 above)
Comment (christain-rogobete) [reply doc/appendix-de-kassensichv/terminology/terminology.md]: Hi Tom, true, thank you for the hint. I removed the dsfivk related changes. Can you pls. check it again?

File: doc/appendix-de-kassensichv/reference-tables/reference-tables.md (hunk 1 above)
Comment (TSchmiedlechner) [reply doc/appendix-de-kassensichv/reference-tables/reference-tables.md]: Looks good to me, thank you!

**Pull request merged**